builder/main.py

```
@@ -515,6 +509,114 @@ def build_fs_image(target, source, env):
+        source_path = Path(source_dir)
+        if source_path.exists():
+            for item in source_path.rglob("*"):
+                rel_path = item.relative_to(source_path)
+                fs_path = "/" + rel_path.as_posix()
+
+                if item.is_dir():
+                    try:
+                        partition.mkdir(fs_path)
+                    except Exception as e:
+                        # Directory might already exist or be root
+                        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)